src/entries/popup/pages/home/Points/ClaimSheet.tsx

```diff
@@ -0,0 +1,199 @@
+  const { amount: zoraBalance } = useUserAssetsBalance({
+    chain: ChainId.zora,
+    currency: 'ETH',
+  });
+  const zoraDisplay = `${parseFloat(Number(zoraBalance).toFixed(8))} ETH`;
+  return (
+    <BottomSheet
+      show
+      onClickOutside={() => navigate(-1)}
+      zIndex={zIndexes.BOTTOM_SHEET}
+    >
+      <Box paddingTop="24px" paddingBottom="12px" isModal>
+        <Box paddingBottom="16px">
+          <Text
+            weight="heavy"
+            size="20pt"
+            color="accent"
+            textShadow="16px accent"
+            align="center"
+          >
+            {i18n.t('points.rewards.choose_claim_network')}
+          </Text>
+        </Box>
+        <Stack gap="10px">
+          <Separator color="separatorTertiary" />
+          <Rows>
+            <Row>
+              <ClaimSheetRow
+                balance={rewards?.claimable || '0'}
+                chain={ChainId.optimism}
+                display={opDisplay}
+              />
+            </Row>
+            <Row>
+              <ClaimSheetRow
+                balance={rewards?.claimable || '0'}
+                chain={ChainId.base}
+                display={baseDisplay}
+              />
+            </Row>
+            <Row>
+              <ClaimSheetRow
+                balance={rewards?.claimable || '0'}
+                chain={ChainId.zora}
+                display={zoraDisplay}
+              />
+            </Row>
+          </Rows>
+        </Stack>
+        <Box
+          paddingHorizontal="20px"
+          paddingTop="24px"
+          paddingBottom="10px"
+          background="surfacePrimaryElevated"
+        >
+          <Button
+            color="fillTertiary"
+            onClick={() => navigate(-1)}
+            width="full"
+            borderRadius="12px"
+            height="44px"
+            variant="transparentShadow"
+            tabIndex={0}
+            paddingHorizontal="20px"
+            disabled={!isSuccess}
+          >
+            <Text
+              size="16pt"
+              weight="bold"
+              textShadow="16px label"
+              color="labelTertiary"
+            >
+              {i18n.t('close')}
+            </Text>
+          </Button>
+        </Box>
+      </Box>
+    </BottomSheet>
+  );
+}
+
+function ClaimSheetRow({
+  balance,
+  chain,
+  display,
+}: {
+  balance: string;
+  chain: ChainId;
+  display: string;
+}) {
+  const navigate = useRainbowNavigate();
+  return (
+    <Inset horizontal="8px">
+      <Box
+        style={{
+          filter: chain !== ChainId.optimism ? 'grayscale(1)' : 'grayscale(0)',
+        }}
+        paddingVertical="10px"
+        className={rowTransparentAccentHighlight}
+        borderRadius="12px"
+        as={motion.div}
+        whileTap={{ scale: 0.98 }}
+        whileFocus={{ scale: 1.02 }}
+        whileHover={{ scale: 1.02 }}
+        paddingHorizontal="8px"
+        onClick={() => {
+          if (chain !== ChainId.optimism) return;
```

Comment:
i see base and zora as chains to be used here, click won't do anything for those cases?

Reply:
discussed offline, other networks are currently unavailable